package.json

```diff
@@ -11,7 +11,7 @@
     "@asyncapi/converter": "^1.1.0",
     "@asyncapi/diff": "^0.4.0",
     "@asyncapi/generator": "^1.9.12",
-    "@asyncapi/modelina": "^0.59.8",
+    "@asyncapi/modelina": "^1.0.0-next.22",
```

Comment:
did you adjust modelina release config, so this is not reverted to `0.59.9` with next regular release?

Reply:
Can I adapt the release workflow when it's managed by .github? 🤔

Reply:
oh crap, you are right, it is possible with react component with https://github.com/asyncapi/asyncapi-react/blob/master/.github/workflows/bump.yml#L34 hack in global workflow but only because HTML template only depends on react component.

but `cli` needs bumps from other packages, not only `modelina` 🤔 

we need something smarter 🤔

Reply:
Opting out `modelina` from `bump.yml` updates pushed from `.github` is the only solution I see I think.
It will require some refactor to https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml config.

other than that, I could always add some feature to `derberg/npm-dependency-manager-for-your-github-org` but not sure what that could be

Reply:
Would it make sense that `derberg/npm-dependency-manager-for-your-github-org` was context depending on the version strategy, and "understand" what to update and what not to? 🤔 I.e. in this case "understand" that because we use a next-generation branch, only updates from that can be used going forward instead of those from master? 🧐

Reply:
in our scenario `bump` runs in scope of modelina. When it runs in `next` branch scope, it surely can figure that out, that it runs in `next` branch, and do something specific. But the problem is when it runs in scope of `master`, how will it know to not do anything in dependant projects? in `cli` it works on default master branch 🤷🏼 

I don't think there is any other solution than:
- have some .bump.config.yml file that you can add to repo, where you can specify what repos should be ignored, and you would store it in master


or

- modify settings of workflow that updates all repos, manage `bump.yml` workflow as separate workflow, independent from `nodejs` topic. Then we can have `autobump` topic. Only repos with this topic would get updates, and `modelina` would be opted out (not only modelina as we have the same case with parser now) from bump workflow updates and could this way have custom settings (like repos to ignore or entire workflow removed from master)

cc @KhudaDad414 @magicmatatjahu @smoya

Reply:
> But the problem is when it runs in scope of master, how will it know to not do anything in dependant projects?

Will it not be able to process the dependent project package.json file to determine the current version and see whether it uses a pre-build id or which major version it uses?

Reply:
technically possible, but why should it have such functionality?

basing on what condition will it know that RC used in package.json is the last one, and actually main release should be used?

Reply:
I might be missing some crucial point here, but in the bump workflow, it should be possible to get the new version of the library, and then take the version of the library it tries to bump, and correlate the semver version with each other to know whether to bump it or not.

Of course, this should not automatically update from pre-release to a new major version. Or I guess it can 🤷